.github/CODEOWNERS

```diff
@@ -0,0 +1,18 @@
+# See https://help.github.com/articles/about-codeowners/
+# for more info about CODEOWNERS file
+
+/tests/async_engine @njhill
+/tests/test_inputs.py @DarkLight1337
+/tests/entrypoints @DarkLight1337 @robertgshaw2-neuralmagic
+/tests/models @DarkLight1337 @ywang96
+/tests/multimodal @DarkLight1337 @ywang96
+/tests/prefix_caching @comaniac
+/tests/spec_decode @njhill @LiuXiaoxuanPKU
+/tests/kernels @tlrmchlsmth
+/tests/quantization @mgoin
+/.buildkite/lm-eval-harness @mgoin
+/tests/distributed/test_multi_node_assignment.py @youkaichao
+/tests/distributed/test_pipeline_parallel.py @youkaichao
+/tests/distributed/test_same_node.py @youkaichao
+/tests/multi_step @SolitaryThinker
+/tests/weight_loading @dsikka
```

Comment:
If we require people with write access then maybe @alexm-neuralmagic can take multi-step and @mgoin (me) can take weight loading

Reply:
updated!